package.json

```diff
@@ -12,12 +12,11 @@
   "scripts": {
     "test": "npm run test:server && npm run test:browser && npm run test:features",
     "test:server": "mocha \"test/unit/**/*-test.js\"",
+    "test:server:coverage": "nyc --reporter=text-lcov npm run test:server | coveralls",
     "test:browser": "mochify \"test/unit/**/*.js\"",
     "test:features": "node scripts/cucumber.js",
-    "test:coverage": "nyc --reporter=text-lcov npm run test:server",
-    "coveralls": "npm run test:coverage | coveralls",
     "ci:lint": "npm run lint",
-    "ci:test": "npm run test && npm run coveralls",
+    "ci:test": "npm run test:server:coverage && npm run test:browser && npm run test:features",
```

Comment:
I duplicate tests pipeline for CI, substituting server test with test+coverage. That looks like a compromise between repeating code and introducing extra complexity. The goal is to execute the same pipeline on CI that you can on local.

Do you think this is a proper approach, @honzajavorek?